readme.txt

```diff
@@ -1,9 +1,9 @@
 === Envato Market ===
 Website: https://www.envato.com/lp/market-plugin/
-Contributors: valendesigns, dtbaker, aaronrutley
+Contributors: valendesigns, dtbaker, aaronrutley, ostigley
```

Comment:
Not necessary for this particular plugin because we self host the update, but in a real WordPress plugin these will link through to a WordPress profile: https://profiles.wordpress.org/ostigley/

Reply:
AH good to know.